apps/cli/src/baseCommand.ts

```diff
@@ -57,9 +57,13 @@ export abstract class BaseCommand<T extends typeof Command> extends Command {
         return ps?.State;
     }
 
+    protected getContextPath(...paths: string[]): string {
+        return path.join(".sunodo", ...paths);
```

Comment:
Could we get the name dynamically like you did wth `${this.config.bin}`, don't know if it is worth but could remove `.sunodo` reference here also.

Reply:
I think it's enough to just have this single point with `.sunodo`.
Maybe we will have to support both `.sunodo` and `.cartesi` in the transition. Subject for another PR anyway.